src/asInput/index.jsx

```diff
@@ -85,10 +85,28 @@ const asInput = (WrappedComponent) => {
 
     render() {
       const { description, error, describedBy } = this.getDescriptions();
-
+      if (order) {
+        return (
+          <div className={styles['form-group']}>
+            <label htmlFor={this.state.id}>{this.props.label}</label>
+            <WrappedComponent
+              {...this.props}
+              {...this.state}
+              className={[
+                styles['form-control'],
+                ...this.props.className,
+              ]}
+              describedBy={describedBy}
+              onChange={this.handleChange}
+              onBlur={this.handleBlur}
+            />
+            {error}
+            {description}
+          </div>
+        );
+      }
```

Comment:
Copy-pasting this block of code is the fastest solution, but that doesn't mean it's the best solution. When you write software, you should strive to keep it as DRY (http://programmer.97things.oreilly.com/wiki/index.php/Don%27t_Repeat_Yourself) as possible. Both this `WrappedComponent` and its label now exist in two places within a few lines of each other. Duplicating data and logic makes future maintenance unnecessarily complicated (a future maintainer would need to update `WrappedComponent` or the <label> in both places). Please store both of these in variables.

Multiple return statements can also be problematic, although it depends how they're used. Typically you use multiple returns if you need to exit out of a loop or a particularly long control chain, but not in a case like this. [This article](https://www.tomdalling.com/blog/coding-tips/coding-tip-have-a-single-exit-point/) gives a pretty good summary of why it's an antipattern. Instead, your if statement here should modify the variable you store `WrappedComponent` in (either appending or prepending the label based on the value of `labelFirst`) so you can return the whole thing in a single return statement at the bottom of this render method.